src/system/unwind.c

```diff
@@ -96,9 +96,18 @@ _Unwind_Ptr __gnu_Unwind_Find_exidx(_Unwind_Ptr pc, int* nrec) {
 	return (_Unwind_Ptr)&__exidx_start;
 }
 
+struct {
```

Comment:
The code style for this struct is wack. Could you please give the struct a typedef (such as trace_t)

Reply:
Implemented this